yjit/src/core.rs

```diff
@@ -8,10 +8,10 @@ use InsnOpnd::*;
 use TempMapping::*;
 
 // Maximum number of temp value types we keep track of
-const MAX_TEMP_TYPES: usize = 8;
+pub const MAX_TEMP_TYPES: usize = 8;
 
 // Maximum number of local variable types we keep track of
-const MAX_LOCAL_TYPES: usize = 8;
+pub const MAX_LOCAL_TYPES: usize = 8;
```

Comment:
Why does this need to be visible outside of `core.rs` ? (or am I missing something?)